packages/react-native-client/src/RNFishjamClientModule.ts

```diff
@@ -80,7 +80,25 @@ type RNFishjamClient = {
   stopForegroundService: () => void;
 };
 
-const nativeModule = requireNativeModule('RNFishjamClient');
-export const nativeModuleEventEmitter = new EventEmitter(nativeModule);
+export const ReceivableEvents = {
+  IsMicrophoneOn: 'IsMicrophoneOn',
+  IsScreenShareOn: 'IsScreenShareOn',
+  IsAppScreenShareOn: 'IsAppScreenShareOn', // only for iOS
+  SimulcastConfigUpdate: 'SimulcastConfigUpdate',
+  PeersUpdate: 'PeersUpdate',
+  AudioDeviceUpdate: 'AudioDeviceUpdate',
+  SendMediaEvent: 'SendMediaEvent',
+  BandwidthEstimation: 'BandwidthEstimation',
+  ReconnectionRetriesLimitReached: 'ReconnectionRetriesLimitReached',
+  ReconnectionStarted: 'ReconnectionStarted',
+  Reconnected: 'Reconnected',
+  Warning: 'Warning',
+  PeerStatusChanged: 'PeerStatusChanged',
+  CurrentCameraChanged: 'CurrentCameraChanged',
+} as const;
 
-export default nativeModule as RNFishjamClient & NativeModule;
+export default requireNativeModule('RNFishjamClient') as RNFishjamClient &
+  NativeModule<
+    // TODO: Make event arguments typesafe instead of generic.
```

Comment:
**note:** Since we now can typecheck all events I'd do it in a separate PR so it's easier to review.